.travis.yml

```diff
@@ -15,18 +15,19 @@ notifications:
 env:
   global: # default values
     - ROS_DISTRO=melodic
-    - ROS_REPO=ros
+    - DOCKER_IMAGE=melodic-ci
```

Comment:
If you go for generalizing the `DOCKER_IMAGE`, please allow/require fully qualified names, i.e. `moveit/moveit:melodic-ci`.